lib/miq_ldap.rb

```diff
@@ -4,6 +4,7 @@ class MiqLdap
   include Vmdb::Logging
   DEFAULT_LDAP_PORT      = 389
   DEFAULT_LDAPS_PORT     = 636
+  IP_REGEXP              = /^(\d{1,3}\.){3}\d{1,3}$/
```

Comment:
Should the callers of this change to use String#ipaddress? from more_core_extensions since that checks IPv4 and IPv6?

Reply:
@bdunne, @djberg96 and @Fryguy Thank you. `String#ipaddress?` 👍